src/test/scala/pt/tecnico/dsi/neutron/NetworksSpec.scala

```diff
@@ -1,22 +1,25 @@
 package pt.tecnico.dsi.neutron
 
-import cats.effect.IO
+import cats.effect.{IO, Resource}
+import org.scalatest.Assertion
+import cats.implicits._
```

Comment:
Codacy found an issue: [Imports should be sorted alphabetically](https://app.codacy.com/gh/ist-dsi/scala-neutron-client/pullRequest?prid=5894722)